backend/spec/requests/comments_spec.rb

```diff
@@ -33,4 +33,60 @@
       end
     end
   end
+
+  describe 'PUT /memos/:memo_id/comments/:id' do
+    context 'コンテンツが有効な場合' do
+      let(:memo) { create(:memo) }
+      let(:comment) { create(:comment, memo: memo) }
+      let(:valid_comment_params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
+
+      it 'コメントが更新され、204になる' do
+        aggregate_failures do
+          put "/memos/#{memo.id}/comments/#{comment.id}", params: { comment: valid_comment_params }, as: :json
+          expect(response).to have_http_status(:no_content)
+          expect(response.body).to be_empty
+        end
+      end
+    end
+  end
+
+  context 'バリデーションエラーになる場合' do
+    let(:memo) { create(:memo) }
+    let(:comment) { create(:comment, memo: memo) }
+    let(:invalid_comment_params) { { content: '' } }
+
+    it 'コメントが更新されていないこと、422になることを確認する' do
+      aggregate_failures do
+        put "/memos/#{memo.id}/comments/#{comment.id}", params: { comment: invalid_comment_params }, as: :json
+        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response.parsed_body['errors']).to eq(['内容を入力してください'])
+      end
+    end
+  end
+
+  context '存在しないコメントIDの場合' do
+    let(:memo) { create(:memo) }
+    let(:comment) { create(:comment, memo: memo) }
+    let(:valid_comment_params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
+
+    it '404が返ることを確認する' do
+      aggregate_failures do
+        put "/memos/#{memo.id}/comments/0", params: { comment: valid_comment_params }, as: :json
+        expect(response).to have_http_status(:not_found)
+      end
+    end
+  end
+
+  context '存在しないメモIDの場合' do
+    let(:memo) { create(:memo) }
+    let(:comment) { create(:comment, memo: memo) }
+    let(:valid_comment_params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
+
+    it '404が返ることを確認する' do
```

Comment:
IMO

itブロックの文言には結果を書くのが慣習的なので、
「確認する」までは書かなくていいかなと思いました！

```suggestion
    it '404が返る' do
```